tests/end_to_end_tests/container_images_test/tools/__init__.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2014-2023 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pathlib as pl
+import subprocess
+from typing import Type
+
+from agent_build_refactored.utils.constants import CpuArch
+from agent_build_refactored.utils.docker.common import delete_container
+from agent_build_refactored.utils.docker.buildx.build import buildx_build, DockerImageBuildOutput
+from agent_build_refactored.container_images.image_builders import (
+    ALL_CONTAINERISED_AGENT_BUILDERS,
+    ContainerisedAgentBuilder,
+    ImageType,
+)
+
+_PARENT_DIR = pl.Path(__file__).parent
+
+
+def build_test_version_of_container_image(
```

Comment:
Add more detailed comment describing how is the test version of the image different.